ocs_ci/ocs/node.py

```diff
@@ -49,7 +53,7 @@ def get_typed_nodes(node_type='worker', num_of_nodes=None):
 
     """
     nodes = get_node_objs()
-
+    assert nodes, "Failed to get the nodes OCS objects"
```

Comment:
As we are already adding assert in `get_node_objs()` itself, do we still need this one?

Reply:
no, again, good catch